packages/modeling/src/operations/expansions/expandGeom3.test.js

```diff
@@ -0,0 +1,32 @@
+const test = require('ava')
+
+const { geom3, poly3 } = require('../../geometries')
+const { expand } = require('./index')
```

Comment:
instead of using 'expand', let's test with expandGeom3

`const expandGeom3 = require('./expandGeom3')`